<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseConversorDialog</class>
 <widget class="QDialog" name="BaseConversorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversor de Base</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Base de Saída:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Converter!</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditInputBase">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEditInputValor"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Saída</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Base de Entrada:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEditOutputValor">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Entrada:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Inverter Entrada/Saída</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEditOutputBase">
      <property name="text">
       <string>16</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="baseTypeInput">
      <property name="text">
       <string>Tipo de base da entrada</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Erros:</string>
      </property>
      <widget class="QLabel" name="labelError">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>351</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QComboBox" name="inputType">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Inteiro Positivo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sinal Magnitude</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Complemento B-1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Complemento B</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="baseTypeOutput">
      <property name="text">
       <string>Tipo de base da saída</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QComboBox" name="outputType">
      <item>
       <property name="text">
        <string>Inteiro Positivo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sinal Magnitude</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Complemento B-1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Complemento B</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
